site/public/js/calendar.js

```diff
@@ -871,3 +871,22 @@ function colorLegend() {
     });
 }
 
+// Modifies cookies so the correct filtering of courses on the calendar is chosen.
+// param string course_val Id of the course
+// param string display_name display name of the course
+function filter_course(course_val, display_name) {
+    if (course_val === 'show all') {
+        Cookies.set('calendar_show_all', '1', { expires: 365 });
+    }
+    else {
+        Cookies.set('calendar_show_all', '0', { expires: 365 });
+        if (display_name) {
+            Cookies.set('calendar_display_name', display_name, { expires: 365 });
+        }
+        else {
+            Cookies.set('calendar_display_name', course_val, { expires: 365 });
```

Comment:
Is this cookie ever used?  A brief search for the string `calendar_display_name` in the codebase didn't turn up any results.